internal/tiltfile/live_update.go

```diff
@@ -0,0 +1,297 @@
+package tiltfile
+
+import (
+	"fmt"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"go.starlark.net/syntax"
+
+	"go.starlark.net/starlark"
+
+	"github.com/windmilleng/tilt/internal/model"
+)
+
+type liveUpdateDef struct {
+	imageName           string
+	steps               []model.LiveUpdateStep
+	fullRebuildTriggers []string
+
+	// whether this has been matched to a deployed image
```

Comment:
```suggestion
	// whether this has been matched to an image we know how to build
```

Reply:
changed to "a deployed image we know how to build"

unfortunately we don't match live updates to builds until we match builds to deploys

Reply:
oh shoot you're right, it DOES need to be a deployed image. nvm.